include/RAJA/util/View.hpp

```diff
@@ -296,6 +297,43 @@ RAJA_INLINE AtomicViewWrapper<ViewType, AtomicPolicy> make_atomic_view(
   return RAJA::AtomicViewWrapper<ViewType, AtomicPolicy>(view);
 }
 
+struct layout_left{};
+struct layout_right{};
+
+template<typename LAYOUT>
```

Comment:
Can we add a specialization for a generic index sequence like std::index_sequence?

Reply:
Sure, I'll take a pass at this and then I'll remove the "Draft" marking from this PR.